ocp_resources/node_network_configuration_policy.py

```diff
@@ -440,6 +440,11 @@ def nnces(self):
                 nnces.append(nnce)
         return nnces
 
+    def node_nnce(self, node_name):
+        return [
+            nnce for nnce in self.nnces if nnce.labels["nmstate.io/node"] == node_name
```

Comment:
@myakove @rnetser I am bit confused here . why we make return output as list. if policy name is known, i should just be getting one enactment (NodeNetworkConfigurationEnactment object. )

Reply:
You are right, I will fix it.